test/hamiltonian/test_heat.py

```diff
@@ -0,0 +1,304 @@
+                + e_zz * qi.Operator.from_label("ZZ")
+                + e_ix * qi.Operator.from_label("XI")
+                + e_iy * qi.Operator.from_label("YI")
+                + e_iz * qi.Operator.from_label("ZI")
+            ).data
+        )
+
+        return generator_ham
+
+    @data(
+        [0.08, -0.01],
+        [-0.05, 0.13],
+        [0.15, 0.02],
+        [-0.04, -0.02],
+        [0.0, 0.12],
+        [0.12, 0.0],
+    )
+    @unpack
+    def test_x_error_amplification(self, e_zx, e_ix):
+        """Test for X error amplification."""
+        exp = ZX90HeatXError(qubits=(0, 1), backend=self.backend)
+        generator = self.create_generator(e_zx=e_zx, e_ix=e_ix)
+        gate = self.create_heat_gate(generator)
+        exp.set_experiment_options(heat_gate=gate)
+        exp.set_transpile_options(basis_gates=["x", "sx", "rz", "unitary"])
+
+        exp_data = exp.run().block_for_results()
+
+        self.assertAlmostEqual(exp_data.analysis_results("A_IX").value.value, e_ix, delta=0.01)
+        self.assertAlmostEqual(exp_data.analysis_results("A_ZX").value.value, e_zx, delta=0.01)
+
+    @data(
+        [0.02, -0.01],
+        [-0.05, 0.03],
+        [0.03, 0.02],
+        [-0.04, -0.01],
+        [0.0, 0.01],
+        [0.01, 0.0],
+    )
+    @unpack
+    def test_y_error_amplification(self, e_zy, e_iy):
+        """Test for Y error amplification."""
+        exp = ZX90HeatYError(qubits=(0, 1), backend=self.backend)
+        generator = self.create_generator(e_zy=e_zy, e_iy=e_iy)
+        gate = self.create_heat_gate(generator)
+        exp.set_experiment_options(heat_gate=gate)
+        exp.set_transpile_options(basis_gates=["x", "sx", "rz", "unitary"])
+
+        exp_data = exp.run().block_for_results()
+
+        # larger error torelance to allow commutator term
+        self.assertAlmostEqual(exp_data.analysis_results("A_IY").value.value, e_iy, delta=0.05)
+        self.assertAlmostEqual(exp_data.analysis_results("A_ZY").value.value, e_zy, delta=0.05)
+
+    @data(
+        [0.02, -0.01],
+        [-0.05, 0.03],
+        [0.03, 0.02],
+        [-0.04, -0.01],
+        [0.0, 0.01],
+        [0.01, 0.0],
+    )
+    @unpack
+    def test_z_error_amplification(self, e_zz, e_iz):
+        """Test for Z error amplification."""
+        exp = ZX90HeatZError(qubits=(0, 1), backend=self.backend)
+        generator = self.create_generator(e_zz=e_zz, e_iz=e_iz)
+        gate = self.create_heat_gate(generator)
+        exp.set_experiment_options(heat_gate=gate)
+        exp.set_transpile_options(basis_gates=["x", "sx", "rz", "unitary"])
+
+        exp_data = exp.run().block_for_results()
+
+        # larger error torelance to allow commutator term
+        self.assertAlmostEqual(exp_data.analysis_results("A_IZ").value.value, e_iz, delta=0.05)
+        self.assertAlmostEqual(exp_data.analysis_results("A_ZZ").value.value, e_zz, delta=0.05)
+
+    @data(123, 456)
+    def test_pseudo_calibration(self, seed):
```

Comment:
Nice. I don't have much sense of the simulator. Does this run reasonably quickly?

Reply:
This is not super quick (but kind of okey for CI). This is why there are only two configurations.